src/v2/Components/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import styled from "styled-components"
+import { BoxProps, Text, TextVariant, boxMixin, color } from "@artsy/palette"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+const Container = styled(RouterLink)<
+  BoxProps & { hasLighterTextColor: boolean }
+>`
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  cursor: pointer;
+  ${boxMixin};
+
+  &:hover,
+  &:focus {
+    outline: none;
+    background-color: ${({ hasLighterTextColor }) =>
+      hasLighterTextColor ? color("black10") : color("black5")};
+  }
+`
+
+interface MenuItemProps
+  extends BoxProps,
+    React.HTMLAttributes<HTMLAnchorElement> {
+  href?: string
+  hasLighterTextColor?: boolean
+  children: React.ReactNode
+  variant?: TextVariant
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+export const MenuItem: React.FC<MenuItemProps> = ({
+  children,
+  href,
+  px = 2,
+  py = 1,
+  hasLighterTextColor,
```

Comment:
Will rename this prop.

Could also do this automatically with something like https://github.com/auspices/eos/blob/b2e8a5067589c364482c3e44cfbef2da82895568/src/theme/utils.ts#L43-L57 but I don't think we're doing anything similar anywhere else